blazor/chart/chart-print.md

```diff
@@ -288,6 +289,109 @@ The optional parameters for this method are,
     };
 }
 
+```
+
+### Export Chart as base64 string
+
+The image can be exported as base64 string for the JPEG, PNG and PDF formats. The rendered Chart can be exported to image as base64 string using the [OnExportComplete](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Charts.ChartEvents.html#Syncfusion_Blazor_Charts_ChartEvents_OnExportComplete) event. The arguments that are required for this method is image type, file name, orientation of the exported PDF document which must be set as null for image export and 0 or 1 for the PDF export and finally allowDownload which should be set as false to return base64 string. 
```

Comment:
Base64 strings for JPEG, PNG, and PDF images can be obtained using the OnExportComplete event. The required arguments for this method include the image type, file name, orientation of the exported PDF document, and setting allowDownload to false in order to retrieve the base64 string.

Add API link